app/assets/javascripts/filter_set.js.coffee

```diff
@@ -0,0 +1,27 @@
+class @FilterSet
+  constructor: (@$root, @key, @val) ->
+
+  $: (selector) ->
+    @$root.find(selector)
+
+  children: ->
+    @$("[data-filter-key=#{ @key }][data-filter-value=#{ @val }]")
+
+  adjacentChildren: ->
```

Comment:
ah interesting - wonder if this might be a good use for `aria-owns` http://tink.uk/using-the-aria-owns-attribute/ ? (truly not sure, just saw that post and thought it was interesting)